### PR TITLE
Adding more commit information to Build

### PR DESCRIPTION
**Pull request by stjohnjohnson**
Including:
 - Who started the build (link to user info)
 - Who authored the commit (link to user info)
 - Commit message
 - Link to Commit

### CODE REVIEW

File: models/build.js [hunk 1]
```diff
@@ -47,14 +47,44 @@ const MODEL = {
...
 
     sha: Joi
         .string().hex()
         .length(40)
         .description('SHA this project was built on')
         .example('ccc49349d3cffbd12ea9e3d41521480b4aa5de5f'),
 
+    startedBy: Joi
+        .string()
+        .description('Identier of who started this build')
```
Comment (Filbird) [inline models/build.js]: Identifier* :^)

**Comment by Filbird** [comment]
👍

**Comment by d2lam** [comment]
Code looks fine. 👍  
But is that too much information? I imagine `startedBy` and `authoredBy` would be enough. Do we really need `startedByUri` and `authoredByUri`? Isn't it just going to be `http://github.com/${username}`

**Comment by stjohnjohnson** [comment]
@d2lam the point of this is so that we don't hard code github.com.  What if you're running your own GHE instance or we want to support GitLab.

**Comment by tkyi** [comment]
👍

**Pull request merged**